syntax = "proto3";

import "boneData.proto";

option optimize_for = LITE_RUNTIME;

message animData_proto
{
	repeated	boneData_proto	bone_Data			= 1;
				bytes			animName			= 2;
				sint32			jointIndex			= 3;
				sint32			parentIndex			= 4;
				uint32			totalAnimFrames		= 5;
}